src/applications/personalization/account/components/MultifactorMessage.jsx

```diff
@@ -1,36 +1,54 @@
 import React from 'react';
-import { mfa } from '../../../../platform/user/authentication/utilities';
-import recordEvent from '../../../../platform/monitoring/record-event';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-function recordAnalyticEvent() {
-  recordEvent({ event: 'multifactor-link-clicked' });
-  mfa();
-}
+import { ssoe } from 'platform/user/authentication/selectors';
+import { mfa } from 'platform/user/authentication/utilities';
+import recordEvent from 'platform/monitoring/record-event';
+
+export class MultifactorMessage extends React.Component {
```

Comment:
This didn't need to be converted from a functional component to a class. Could have just added `useSSOe` to the arg passed to `MultifactorMessage()`

Reply:
thanks, I'll revert that back to a function

Reply:
@erikphansen to do this will I need to change the caller, AccountMain, to be a connected component?

Reply:
mmm no. `useSSOe` will come from the new `mapStateToProps()` so it should just work to go back to `MultifactorMessage({ multifactor, useSSOe })`

Reply:
🆒